src/app/tests/integration/chip_im_initiator.cpp

```diff
@@ -48,9 +48,12 @@ constexpr size_t kMaxCommandMessageCount          = 3;
 constexpr size_t kTotalFailureCommandMessageCount = 1;
 constexpr size_t kMaxReadMessageCount             = 3;
 constexpr size_t kMaxWriteMessageCount            = 3;
-constexpr uint32_t gMessageIntervalMsec           = 1200;
-constexpr uint32_t gMessageTimeoutMsec            = 1000;
-constexpr chip::FabricIndex gFabricIndex          = 0;
+
+constexpr uint32_t gMessageIntervalMsec   = 1200;
+constexpr uint32_t gMessageTimeoutMsec    = 1000;
+constexpr size_t kMaxSubMessageCount      = 1;
+constexpr int32_t gMessageIntervalSeconds = 1;
```

Comment:
please put the new ones on the bottom so it's easier to see what changed

Reply:
Done, thanks